src/components/Calculator.js

```diff
@@ -1,52 +1,57 @@
-import { useState } from 'react';
```

Comment:
Please be sure that your the `App.js` component is class-based as well because its part of the requirements 🤓